### ISSUE TITLE
Form elements lack labels

### ISSUE THREAD

**Issue by torgeirl** [devilry/trix2#84]
Success Criteria 1.3.1 and 3.3.2 in WCAG 2 requires labels to ensure that form controls are announced properly by assistive technologies, like screen readers.

Failing elements:
```HTML
<input type="checkbox" ng-model="negative" class="ng-pristine ng-untouched ng-valid ng-empty">
<select ng-model="tagToAdd" ng-change="addTag()" class="ng-pristine ng-untouched ng-valid ng-empty">
```
